services/woocommerce_importer.py

```
@@ -0,0 +1,480 @@
+                    url,
+                    json={
+                        "chat_id": self.telegram_chat_id,
+                        "text": message
+                    }
+                )
+                response.raise_for_status()
+
+            logger.info("Telegram notification sent", extra={"message": message})
+            return True
+
+        except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)